### DISCUSSION TITLE
麻烦把我误交的会员费退给我，谢谢！

### DISCUSSION THREAD

**Discussion by nanke222-cn** [getlantern/lantern-forum-cn#113] [Q&A]
使用蓝灯应该也有好几年了，付费VPN我只用过蓝灯，也算是铁杆用户了，但是最近这一个月的使用体验实在糟糕！首先是2023年12月30日，我的会员到期，于是选择两年续费，但是输入信用卡信息后，蓝灯APP上提示未支付成功（或是其他什么未成功的提示？我记不清了），于是我再次输入信用卡信息。结果发现手机收到了两条信用卡授权冻结信息，也就是说我被扣了两次钱，每次617元，合计1234元。后面我看我的会员到期时间到了2028年，看来蓝灯是默认我充了两次两年的会员。我尝试了在蓝灯反馈问题那里发送报告，但未收到任何回应，我尝试把情况反馈给我能找到的各个蓝灯的邮箱，但是邮件都无法发出（有可能是邮箱地址不对？我确实不大懂）。我凑合着继续用，结果1月下旬就遇到了长达一周多的服务无法使用。今天下午，我的一个手机终于可以正常使用蓝灯，但另一个手机至今无法使用。我现在请你们给我一个确定可以正常沟通的联系方式，我把交易信息发给你们，然后你们把我其中一个两年会员的费用退还给我。另一个我已经在使用了，我愿意继续使用。谢谢。

**Comment by MaxMKMax** [comment]
@nanke222-cn 

您好。谢谢您的反馈。非常抱歉最好的不稳定影响了您的体验感。好的，我帮您安排退款其中一个(两年)的费用。所以只续费到2026对吧。请您提供我您的邮箱和第二次的付费单号。我帮您安排一下。


我们最近已经加班努力的修复目前的问题了。在近期就可以解决了。对了，我们的邮箱是support@getlantern.org

您可以帮我一个忙吗？您说的无法发出邮件是被blocked吗？您用qq还是gmail, hotmail? 您使用的是哪一个邮箱和用哪一款手机或电脑发邮件的? 安卓?IOS? Window? 还是哪个系统?

我们需要您提供细节来修复这个问题。谢谢您。

**Reply by nanke222-cn** [reply]
我的邮箱是261070015@qq.com，刚刚也通过这个邮箱把交易截图发给你们邮箱了，这次似乎发送成功。之前用的是win7系统

**Reply by nanke222-cn** [reply]
Message blocked
Your message to support@lantern.jitbit.com has been blocked. See technical details below for more information.
The response from the remote server was:
554 5.7.1 <[261070015@qq.com](mailto:261070015@qq.com)>: Sender address rejected: Access denied

刚刚邮箱又提示这个

**Reply by nanke222-cn** [reply]
这种应该怎么处理啊？几天了，麻烦给个回复

**Reply by wangzissss** [reply]
付费邮箱:[891692309@qq.com](mailto:891692309@qq.com) ,一直没有收到25位激活码，请帮忙一下
![Uploading QQ图片20240618104635.png…]()

**Comment by MaxMKMax** [comment]
@nanke222-cn 感谢您为我们提供了一些重要的信息。我会把这个问题和技术团队讨论. 

对了, 您收到其中两年的退款了吗? 技术团队和我说他们已经帮您弄好了. 所以更新时到 2026 年.

**Reply by futurefuturehx** [reply]
你好，我也是这种情况，我的邮箱是 360574435@qq.com，我只想续 1 年的，结果说网络错误，导致我连续操作了 3 次，扣了三年，麻烦帮我改成一年，并退回我其余 2 年的钱

**Reply by nanke222-cn** [reply]
没有，还是2028，也没收到退款。我退款账号尾号是6918，你们确定已经处理了？
![Uploading mmexport1707195416757.jpg…]()

**Reply by 3snail** [reply]
我也遇到这个问题了，我的邮箱是735918519@qq.com，付款时间是2024/02/29 16:50:09 340.00 ~2024/02/29 16:49:54 340.00 ~，麻烦帮我退回一年费用，谢谢

**Reply by wangzissss** [reply]
付费邮箱:[891692309@qq.com](mailto:891692309@qq.com) ,一直没有收到25位激活码，请帮忙一下 @Mike777Khoo 
![QQ图片20240618104635](https://github.com/getlantern/lantern-forum-cn/assets/166413677/62a58d17-bb28-4383-ae3b-c5f5caf18759)

**Comment by MaxMKMax** [comment]
@nanke222-cn 您好. 我之前已经帮你们的问题提交给支付部门了. 我来帮你们追一下, 给我两天时间. 不好意思让你们久等了. 谢谢!

**Comment by MaxMKMax** [comment]
@futurefuturehx 您好. 我之前已经帮你们的问题提交给支付部门了. 我来帮你们追一下, 给我两天时间. 不好意思让你们久等了. 谢谢!

**Comment by MaxMKMax** [comment]
@futurefuturehx 您好。我们的付款团队找不到您的付款记录。您可以在此处分享您的付款单号以及付款或收据的屏幕截图吗？谢谢您!
360574435@qq.com , 您提供了正确的电子邮件吗？

**Reply by futurefuturehx** [reply]
![Snipaste_2024-02-18_09-56-36](https://github.com/getlantern/lantern-forum-cn/assets/22697672/d00e77e1-f1b3-465d-8dcd-0899c75ad7b5)

**Reply by futurefuturehx** [reply]
邮箱对的呀

**Reply by futurefuturehx** [reply]
![1af1de496aa13da2f357637d3f07d8d](https://github.com/getlantern/lantern-forum-cn/assets/22697672/1070ed44-55a2-471d-9f79-13adb8dcc94d)

**Reply by futurefuturehx** [reply]
app错误，导致点了3次

**Comment by MaxMKMax** [comment]
@nanke222-cn 我们的付款部门要求我与您核实一下，您在这一天付款了吗？December 30th, 2023 
我们想确认一下. 谢谢您.

**Reply by nanke222-cn** [reply]
是的，信用卡额度冻结时间是在2023年12月30日，最后在系统查到的具体支付时间是2024年1🈷2日。

**Reply by MaxMKMax** [reply]
@nanke222-cn 好的. 我帮您和支付团队反馈一下.

**Reply by nanke222-cn** [reply]
你好，我的问题你们支付团队什么时候能解决啊？

**Reply by nanke222-cn** [reply]
？？？现在直接不回复了吗？

**Reply by MaxMKMax** [reply]
@nanke222-cn 对不起。我错过了你的消息。我马上帮你追问一下!

**Reply by MaxMKMax** [reply]
@nanke222-cn 这个GitHub太难搞了. 有时会看漏信息. 非常抱歉啊!

**Reply by nanke222-cn** [reply]
好的，麻烦尽快解决吧。主要是又没法给你们发邮件，只有这里还可以沟通

**Reply by MaxMKMax** [reply]
<img width="266" alt="image" src="https://github.com/getlantern/lantern-forum-cn/assets/155475142/2458ddaf-5c08-414c-87b4-52d8e4c4e35d">
@nanke222-cn 你好。我们的付款团队通知我，错误的付款已成功退款。 2028年不再付款，而是2026年。您能在接下来的三天内帮忙查看退款是否成功吗？如果没有成功完成，请告诉我。谢谢。

**Reply by nanke222-cn** [reply]
目前我没收到退款，会员的到期时间也没有变化。我支付会员费的信用卡尾号为6918，麻烦再核实一下

**Reply by MaxMKMax** [reply]
@nanke222-cn 你好。好的。我已经给负责人发了短信，但由于美国时区，他们只会在今天午夜之前上线。他们将帮助您再次检查这个问题。别担心，谢谢。

**Comment by MaxMKMax** [comment]
@nanke222-cn 
<img width="664" alt="image" src="https://github.com/getlantern/lantern-forum-cn/assets/155475142/3de9d267-3f83-4c06-9bc2-ef9a91024ef8">
你好，已经退款了。一切都在进行中。

**Reply by nanke222-cn** [reply]
收到退款了，谢谢

**Reply by MaxMKMax** [reply]
@nanke222-cn 您好. 不客气! 请您以后再回来支持蓝灯. 只是最近网络管得比较严所以影响了大部分的VPN. 希望您能理解. 以后欢迎回来!

**Comment by liusky467-jpg** [comment]
我今天购买的根本用不了，麻烦给退一下款，邮箱是3278881588@qq.com,
我把电子账单给你发过去了，你们也没理我，实在不行只能举报了

![Uploading 微信图片_20251220214723.jpg…]()

**Reply by liusky467-jpg** [reply]
@MaxMKMax @futurefuturehx

**Reply by david-lant** [reply]
您好，关于退款的事情，请通过App，[在线表格](https://support.lantern.io/)或者邮箱[support@getlantern.org](mailto:support@getlantern.org)联系我们。

---

**我们将在12/24停止使用论坛或者Issues回答问题，详情请看[这里](https://github.com/getlantern/lantern-forum-cn/discussions/1540)**。

如果您有问题的话，请使用以下通道解决：
- (推荐）App中的提交问题（因为我们可以看到诊断报告）
- [在线表格](https://support.lantern.io/)
- 邮箱[support@getlantern.org](mailto:support@getlantern.org)
  
如果您通过在线表格或邮箱提交工单，麻烦参照[这里](https://github.com/getlantern/guide/wiki/Troubleshooting#log-retrieval)附件诊断日志log。网络或App问题的话，我们需要诊断log才能解决。

**Reply by liusky467-jpg** [reply]
给你发送邮件也没理我，能不能快点解决？退款就不行吗？

**Reply by liusky467-jpg** [reply]
@MaxMKMax @futurefuturehx

**Reply by liusky467-jpg** [reply]
能不能解决了？你们办事效率就这么低

**Reply by david-lant** [reply]
您的工单号是多少？

**Reply by liusky467-jpg** [reply]
在哪看工单号 @

**Reply by david-lant** [reply]
您发邮件后会收到自动回复，这六位数就是工单后
<img width="175" height="47" alt="Screenshot 2025-12-24 at 16 58 12" src="https://github.com/user-attachments/assets/9598d498-ccf1-4e1b-8404-7363bf9761e3" />

**Reply by liusky467-jpg** [reply]
164322

**Reply by liusky467-jpg** [reply]
可以了吗

**Reply by liusky467-jpg** [reply]
在吗

**Reply by david-lant** [reply]
您好，不好意思我们刚放假回来。正在内部处理。

**Comment by liusky467-jpg** [comment]
啥时候处理啊？一个月了你们

David ***@***.***>于2026年1月6日 周二03:17写道：

**Reply by david-lant** [reply]
您好，刚给您发了邮件，请查收

**Reply by liusky467-jpg** [reply]
没有收到退款啊 @nanke222-cn @3snail @futurefuturehx @MaxMKMax @wangzissss

**Reply by liusky467-jpg** [reply]
都几个月了，还是没有退款
